src/blocks/scratch3_looks.js

```diff
@@ -484,27 +492,36 @@ class Scratch3LooksBlocks {
         );
     }
 
+    clampEffect (effect, value) {
+        let clampedValue;
```

Comment:
For effects other than "ghost" and "brightness", this code should return the value as passed instead of returning `undefined`. Shouldn't this be

```suggestion
        let clampedValue = value;
```

Reply:
Oh right, that's a good catch. Thank you @joker314!

Reply:
I'm surprised forgetting to return any value for `clampedValue` didn't break any tests. I'll add that scenario to my test for this PR as well. Thanks again!